src/main/java/org/broadinstitute/hellbender/tools/DumpTabixIndex.java

```diff
@@ -0,0 +1,166 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import picard.cmdline.programgroups.DiagnosticsAndQCProgramGroup;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.GZIPInputStream;
+
+@CommandLineProgramProperties(
```

Comment:
Add a docstring with explanation and usage, and annotate as a Barclay `@DocumentedFeature`